core/src/main/java/com/graphhopper/routing/ev/Country.java

```diff
@@ -242,28 +242,28 @@ public enum Country {
     ZMB("Zambia", "ZM"),
     ZWE("Zimbabwe", "ZW");
 
-    public static final String KEY = "country";
+    public static final String KEY = "country", JSON_AREA3 = "ISO3166-1:alpha3";
 
-    private final String name;
-    private final String twoLetterCode;
+    private final String displayName;
```

Comment:
`displayName` sounds like you need to distinguish the `name` and a `displayName`? But there is no `name`? Ok, maybe this String is the displayName of the country code rather than the name of the country, but then the enum should be called `CountryCode`... Anyway, my vote goes for `Country` and `name` :)

Reply:
Ah, I see. Maybe `getCountryName()`?